examples/edge-functions/supabase/functions/slack-bot-mention/slack-bot-mention.mdx

```diff
@@ -0,0 +1,80 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'slack-bot-mention',
+  title: 'Slack Bot Mention Edge Function',
+  description: 'An edge function for processing Slack bot mentions.',
+}
```

Comment:
Any reason we use MDX instead of a `README.md` here? I'm not aware of our docs pulling in MDX files from this directory (please correct me if I'm wrong).

Reply:
I think that I am used with the mdx on the other directories and didn't realize that here it was different. Commited your other suggestions.